src/features/donation/models/effects/ft.ts

```
@@ -0,0 +1,218 @@
+        { args: { account_id: referrerAccountId } },
+      )
+    : null;
+
+  const [
+    ftMetadata = null,
+    ftStorageBalanceBounds = null,
+    protocolFeeRecipientFtStorageBalance = null,
+    donationContractFtStorageBalance = null,
+    recipientFtStorageBalance = null,
+  ] = await Promise.all([
+    tokenClient.view<{}, FungibleTokenMetadata>("ft_metadata"),
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)